src/components/card/Card.tsx

```diff
@@ -0,0 +1,158 @@
+import { ThemeColors, useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ID } from 'app/store/notifications/types'
+import { font } from 'app/styles'
+
+export type CardType = 'user' | 'collection'
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    shadowContainer: {
+      elevation: 2,
+      shadowColor: 'rgb(133,129,153)',
+      shadowOpacity: 0.25,
+      shadowOffset: { height: 2, width: 0 },
+      shadowRadius: 2
+    },
+    cardContainer: {
+      backgroundColor: themeColors.white,
+      borderRadius: 8,
+      overflow: 'hidden',
+      width: '100%',
+      minWidth: 172,
+      maxWidth: 190
+    },
+    cardContent: {
+      display: 'flex',
+      paddingHorizontal: 8
+    },
+    imgContainer: {
+      paddingTop: 8,
+      paddingHorizontal: 8
+    },
+    cardImg: {
+      position: 'relative',
+      backgroundColor: '#ddd',
+      borderRadius: 6,
+      overflow: 'hidden',
+      paddingBottom: '100%',
+      width: '100%'
+    },
+    userImg: {
+      borderRadius: 1000
+    },
+    textContainer: {
+      paddingVertical: 4
+    },
+    primaryText: {
+      ...font('bold'),
+      color: themeColors.neutral,
+      fontSize: 14,
+      lineHeight: 18,
+      marginBottom: 2,
+      maxWidth: 160,
+      overflow: 'hidden',
+      textAlign: 'center'
+    },
+    secondaryText: {
+      ...font('medium'),
+      color: themeColors.neutral,
+      fontSize: 12,
+      lineHeight: 18,
+      marginBottom: 2,
+      overflow: 'hidden',
+      textAlign: 'center',
+      width: '100%'
+    }
+  })
+
+export type CardProps = {
+  id: ID
+  imageSize: ProfilePictureSizes | CoverArtSizes | null
+  onPress: () => void
+  primaryText: string
+  secondaryText?: string
+  style?: StyleProp<ViewStyle>
+  type?: CardType
+  user: User
+}
+
+const CardImage = ({ id, type, imageSize }) => {
+  const useImage =
+    type === 'user' ? useUserProfilePicture : useCollectionCoverArt
+  const image = useImage(id, imageSize, SquareSizes.SIZE_150_BY_150)
+
+  return <DynamicImage image={{ uri: image }} />
+}
+
+export const Card = ({
+  id,
+  imageSize,
+  onPress,
+  primaryText,
+  secondaryText,
+  style,
+  type = 'user',
+  user
+}: CardProps) => {
+  const styles = useThemedStyles(createStyles)
+
+  return (
+    <View style={[styles.shadowContainer, style]}>
+      <View style={styles.cardContainer}>
+        <TouchableOpacity onPress={onPress}>
+          <View style={styles.cardContent}>
+            <View style={styles.imgContainer}>
+              <View style={[styles.cardImg, type === 'user' && styles.userImg]}>
+                <CardImage imageSize={imageSize} type={type} id={id} />
+              </View>
+            </View>
+            <View style={styles.textContainer}>
+              <Text numberOfLines={1} style={styles.primaryText}>
+                {primaryText}
+                {type === 'user' ? (
+                  <UserBadges
```

Comment:
Does RN not complain about this? Might be better to have the UserBadge outside of the Text component

Also I added the `hideName` prop to UserBadges but it doesn't really sit right with me that we are showing the name in 2 different ways. The UserBadges component in the client doesn't include the name. Thoughts?

Reply:
I agree, I feel like the component should only be responsible for showing the badges